app/components/contact-page-components/SuccessScreen.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Image from "next/image";
+
+import { temporaryHeroIcon } from "@/public/png-icons";
+interface SuccessScreenProps {
+  setShowSuccess: React.Dispatch<React.SetStateAction<boolean>>;
+}
+
+const SuccessScreen: React.FC<SuccessScreenProps> = ({ setShowSuccess }) => {
+  const handleChildClick = (
+    e: React.MouseEvent<HTMLDivElement, MouseEvent>
+  ) => {
+    e.stopPropagation();
+  };
+
+  return (
+    <div
+      className="fixed z-30 flex h-screen w-screen justify-center bg-black/30 px-4 pt-20 md:pt-28"
+      onClick={() => setShowSuccess(false)}
+    >
+      <div
+        className="z-40 flex h-fit w-full max-w-xl flex-col items-center justify-between rounded-xl bg-white900 p-6 dark:bg-black300 md:p-9"
+        onClick={handleChildClick}
+      >
+        <p className="text-center text-2xl font-semibold text-black200 dark:text-white900 md:text-3xl">
+          Thank you for getting in contact
+        </p>
+        <Image
+          src={temporaryHeroIcon}
+          alt="icon of me working"
```

Comment:
Suggestion only... what you have is fine! You could also consider `alt="Illustration of me working at a desk with a computer"`

Reply:
much more descriptive. thanks for the tip